PWE.html

```diff
@@ -30,61 +32,118 @@
-      may ban an individual from participating in our activities, e.g. on a
-      mailing
-      list or in a group (refer to <a href="/Consortium/Process/policies.html#member-rep">section
-        3.1.2 of the W3C
-        Process Document</a>). </p>
-    <h3><a name="ombuds" id="ombuds">Ombuds</a></h3>
-    <p>The W3C Ombuds are a small group selected by the W3C management to act as
-      trusted confidants to work toward resolving complaints between any
-      internal or external constituent. Ombuds will operate by procedures appropriate to
+                       </script>
+    </head>
+    <body>
+        <h1>Positive Work Environment Home Page</h1>
+        <p>This document is a companion to the W3C Code of Conduct. It describes the procedures for handling incidents relating to Code of Conduct violations or complaints.</p>
+    <section id="sotd"></section>
+
+        <section id="introduction">
+            <h2>Introduction</h2>
+<p>As we engage with one another, incidents can happen. Incidents can take many forms, but participants have the right to a safe and positive work environment. The goal of this incident resolution process is that participants in W3C can:</p>
+
+            <ul>
+                <li>Get help and support to safely resolve issues that negatively affect them</li>
+                <li>Correct behaviours that negatively affect other participants in W3C</li>
+                <li>Continue to safely work and contribute to the W3C after an issue has been raised</li>
+            </ul>
+                
+            <p>The Code of Conduct is the standard that all W3C participants are required to follow and is used as the basis for any resolution and disciplinary actions. In exceptional circumstances, individuals may have their W3C participation suspended or withdrawn as a result of a complaint or investigation.</p>
+                
+            <p>All situations are different and W3C participants are encouraged to use whichever method of incident resolution they feel comfortable with.</p>
+                
+            <p>Participants are encouraged to try and resolve issues themselves in the first instance. However, if participants do not feel comfortable or safe doing so, they can get help through an ombudsperson, facilitators and/or mediation.</p>
+                
+            <p>Depending on the severity of the situation, it may not be appropriate to attempt to resolve the issue informally or in the context of the group. Participants should immediately report an incident whenever they feel unsafe or threatened by the behaviour or actions of other W3C participants.</p>
+                
+            <p>Everyone involved in the resolution process is expected to:</p>
+            
+            <ul>
+                <li>Show respect for others</li>
+                <li>Work together to resolve the complaint</li>
+                <li>Maintain confidentiality</li>

```

Comment:
As noted in https://github.com/w3c/PWETF/issues/359 , we need to clarify who should maintain confidentiality, because not everyone needs to maintain confidentiality.  For example, witnesses and those involved should not be expected to maintain confidentiality unless they have separately agreed to do so.